src/building/how-to-build-and-run.md

```diff
@@ -252,4 +250,5 @@ everything up then you only need to run one command!
 ./x.py clean
 ```
 
-`rm -rf build` works too, but then you have to rebuild LLVM.
+`rm -rf build` works too, but then you have to rebuild LLVM, which can take
+a long time (on the order of a half-hour).
```

Comment:
I don't like saying half hour because it can be a lot more or a lot less depending on your computer. "Long time" seems fine, you'll know after the first time you build it.

Reply:
I just wanted to provide a *very* rough timescale. Is there a way to convey that without providing a specific time?

Reply:
'a little longer than a stage 1 build'?

Reply:
The reason I don't think this makes sense is some people (like `@Mark-Simulacrum`) can build from scratch in 5 minutes, while others like `@mark-i-m` need over 2 hours. So there's no time we can give that would be anywhere _close_ for both of them.

Reply:
Wow, I want `@Mark-Simulacrum`'s computer ;)

Reply:
What about this?

```suggestion
a long time even on fast computers.
```

My computer is quadcore and it still takes 30 minutes.

Reply:
Yeah, that sounds reasonable.